app/src/main/java/com/sdpteam/connectout/profileList/FilteredProfileListFragment.java

```diff
@@ -0,0 +1,75 @@
+package com.sdpteam.connectout.profileList;
+
+import android.os.Bundle;
+import android.os.Handler;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.EditText;
+
+import androidx.fragment.app.Fragment;
+
+import com.sdpteam.connectout.R;
+import com.sdpteam.connectout.profile.ProfileFirebaseDataSource;
+import com.sdpteam.connectout.profileList.filter.ProfileFilter;
+import com.sdpteam.connectout.profileList.filter.ProfileNameFilter;
+
+public class FilteredProfileListFragment extends Fragment {
+
+    private final ProfileFilter baseFilter;
+    private final Handler searchHandler = new Handler();
+    private Runnable searchRunnable;
+
+    public FilteredProfileListFragment() {
+        this(ProfileFilter.NONE);
+    }
+
+    public FilteredProfileListFragment(ProfileFilter baseFilter) {
+        this.baseFilter = baseFilter;
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View contentView = inflater.inflate(R.layout.fragment_filter, container, false);
```

Comment:
But here there could be an error as well (not sure at all)
From what was done previously (see file that you deleted in this PR : EventParticipantsListFragment for instance) we provide here the layout which is the xml file name not a xml child like here.

Reply:
For now it works, we can investigate further I agree